Tests/Acceptance.Config.json

```diff
@@ -40,5 +40,7 @@
         }
     ],
     "sqlServerNotificationRecipient": "noreply@devops.com",
-    "suffix": "812d5bex"
-}
\ No newline at end of file
+    "suffix": "812d5bepj",
+    "TagConfigPath" :"..\\Resources\\TagConfig.json"
```

Comment:
This file should probably be put with the tests or it could end up being packaged with the scripts

Reply:
OK

Reply:
I see you have created Test/TagConfig.json, this path needs adding to the Acceptance.config.json file